README.md

```diff
@@ -15,10 +15,35 @@ To see what's incoming, check the [open pull requests](https://github.com/greene
 Instructions for using Manubot Rootstock for your own manuscript are still evolving.
 The recommended approach is to clone this repository, as detailed [here](https://github.com/greenelab/manubot-rootstock/issues/6#issuecomment-314541837).
 
-## Source
+## Repository directories & files
 
-The manuscript source is located in [`content`](content).
-Text should be written in markdown files, with a digit prefix for ordering (e.g. `01.`, `02.`, etc.) and a `.md` suffix.
+See [`CONTRIBUTING.md`](CONTRIBUTING.md) for documentation on how to create or contribute to a manuscript.
```

Comment:
I think fewer files the better.

`USAGE.md` is more accurate than `CONTRIBUTING.md`. Note that GitHub has [integration](https://github.com/blog/1184-contributing-guidelines) with `CONTRIBUTING` files. Whenever a user opens a new issue or PR, they will see the link the contributing docs.

Is this good or bad? Perhaps it's a bit misleading to label usage guidelines and contributing guidelines?